### DISCUSSION TITLE
How to recover harbor after delete all db table records?

### DISCUSSION THREAD

**Discussion by microyahoo** [goharbor/harbor#19896] [Q&A] [answered]
How to recover harbor after delete all db table records?
![screenshot-20240123-191724](https://github.com/goharbor/harbor/assets/10202906/fdcca520-36fd-4bff-a45a-7371109c46e6)

**Comment by chlins** [comment, marked as answer]
You may need to clean the environment and re-install your harbor instance if db data destroyed.

**Reply by microyahoo** [reply]
OK, thanks.
